additional_tests/exchanges_tests/abstract_authenticated_exchange_tester.py

```diff
@@ -1003,6 +1003,21 @@ async def _get_bundled_orders_stop_take_profit(self, current_price):
             take_profit,
         )
 
+    async def test_fetch_positions(self):
+        # pass if not implemented
+        async with self.local_exchange_manager():
+            await self.inner_test_fetch_positions()
```

Comment:
is it a different type of test compared to the positions tests in AbstractAuthenticatedFutureExchangeTester ? If yes, can we give a more explicit name to this test to avoid confusion with positions-releated tests of the futures exchanges test framework ?